src/test/unittest/requirements.py

```diff
@@ -36,6 +36,17 @@ def _check_is_admin(self):
         else:
             return os.getuid() == 0
 
+    def check_ndctl(self):
+        is_ndctl = self._check_pkgconfig('libndctl', NDCTL_MIN_VERSION)
+        if not is_ndctl:
+            raise futils.Skip('libndctl (>=v{}) is not installed'
+                              .format(NDCTL_MIN_VERSION))
+
+    def check_ndctl_enable(self):
```

Comment:
this needs to change to actually detect if the binary is built with ndctl. See `is_ndctl_enabled` in the bash framework.